ibis/expr/types/structs.py

```diff
@@ -99,12 +80,23 @@ def struct(
 
 @public
 class StructValue(Value):
-    """A struct literal or column.
+    """A Struct is a nested type with ordered fields of fixed types.
+
+    Corresponds to
+    - Struct in DuckDB
+    - Struct in BigQuery
+    - Struct in Spark
+    - Struct in Impala
+    - Struct in Apache Arrow
+    - CompositeType in PostgreSQL
+    - etc
```

Comment:
I'm not sure what the value of having these lists of backend types in Ibis's expression type docstrings is. I don't think the specific backend type is relevant for most users.

Reply:
I was thinking that the verbose, precise definition of "a nested type with ordered fields of fixed types." is a little hard to grok. I wanted analogies, assuming that a user would be familiar with at least one of these things. It's not needed after the initial grokking, but helps with that. Open to removing these bits though if you still want to.

Reply:
I think this is probably too much information to throw at a user in a docstring.

I suspect something like a high level description of what a struct is suffices. Thoughts?

Reply:
Yeah I am +0 on this so lets do it.

Reply:
I removed these "similar to" examples from the docstrings of array, struct, and map